modules/comment/comment.service.js

```diff
@@ -53,8 +63,13 @@ class CommentsService {
 
   async addRate(id, data, user) {
     const product = await Product.findById(id);
```

Comment:
Use the `lean` method here in order to reduce the response body since we use product data no more here